### ISSUE TITLE
[CLIFF] upper body reconstruction

### ISSUE THREAD

**Issue by sangdv** [huawei-noah/noah-research#151]
We have a research project related to 3D reconstruction of upper body part to estimate the intubation difficulty of patients. However, when we applied your model on images that contain only upper body part, the model yielded very bad results since it tried to reconstruct the whole human body. Could you please advise us how to deal with that?
Here is an example of an input image with only upper human body part: https://upload.wikimedia.org/wikipedia/commons/f/f8/Upper_body_front.png

**Comment by sangdv** [comment]
Hi, here are my results. I also attach some test images so that you can check on your side. Thanks.

**Comment by sangdv** [comment]
No, we want to reconstruct the upper body including the head, neck and maybe shoulders as well. We then detect some 2D keypoints like the eyes, chin, thyroid notch, sternal notch... and lift them to the 3D keypoints using reconstructed mesh. The purpose is to calculate some distances between them as clinical features to estimate difficult intubation.

**Comment by sangdv** [comment]
Here are some other examples: https://drive.google.com/drive/folders/14HFcTCRSFMLeuMXYvTJ4Hds6HDzhccbD
Can you let me know how to fit the 3D shape better? The 3D reconstructed shape seems to be quite small compared to the original one.

**Comment by zhihaolee** [comment]
Please refer to this repo: https://github.com/akashsengupta1997/STRAPS-3DHumanShapePose.

**Comment by sangdv** [comment]
Thank you for your suggestion. I tried their improved method at https://github.com/akashsengupta1997/HierarchicalProbabilistic3DHuman but the results are still the same as you can see here: https://drive.google.com/drive/folders/1_Dt7nGL0-lBzG59z82-x1LkcOfjmm95U?usp=sharing

**Comment by zhihaolee** [comment]
I don't have the right to access the google drive data.
Maybe you can try https://github.com/muelea/shapy, and see how it goes.
